apache.yml

```diff
@@ -6,6 +6,8 @@ services:
             - ./config/apache/sites-available/001-dynamic-vhost-ez5.conf:/etc/apache2/sites-available/001-dynamic-vhost-ez5.conf
             - ./config/apache/sites-available/002-dynamic-vhost-ezplatform.conf:/etc/apache2/sites-available/002-dynamic-vhost-ezplatform.conf
             - ./config/apache/sites-available/003-dynamic-vhost-ez4.conf:/etc/apache2/sites-available/003-dynamic-vhost-ez4.conf
+            - ./config/apache/sites-available/004-dynamic-vhost.conf:/etc/apache2/sites-avalaible/004-dynamic-vhost.conf
+            - ./config/apache/sites-available/004-dynamic-vhost.conf:/etc/apache2/sites-enabled/004-dynamic-vhost.conf
```

Comment:
To be removed & add a2ensite in bootstrap.sh for this vhost in apache images (5.4/5.6/7.0)